src/server/ua_services_nodemanagement.c

```diff
@@ -1,6 +1,21 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. 
+ *
+ *    Copyright 2014 (C) Leon Urbas
+ *    Copyright 2014-2016 (C) Sten Grüner
+ *    Copyright 2014-2017 (C) Florian Palm
+ *    Copyright 2014-2017 (C) Julius Pfrommer
+ *    Copyright 2015-2016 (C) Chris Iatrou
+ *    Copyright 2015-2016 (C) Oleksiy Vasylyev
+ *    Copyright 2016 (C) LEvertz
```

Comment:
He worked on it while at acplt. Perhaps this could have his first name, too.
I can contact him, if this is important enough.